sbol_utilities/package.py

```diff
@@ -94,26 +95,33 @@ def define_package(doc: sbol3.Document):
         # Namespace must match the hasNamespace value for all of its members
         package.namespace = package_namespace
 
+        # Combine all of the docs into one
+        doc = args[0]
+        for sbol_doc in args[1:]:
+            doc_objects = [o for o in sbol_doc.objects]
```

Comment:
Copying the objects is making a build artifact for the package (see [this section in SEP 054](https://github.com/SynBioDex/SEPs/blob/master/sep_054.md#collating-a-native-package-build-artifact)), which we will also want to have a function for - you can break this out into that function.  

For defining a package, we just need to take the inventory of each of the sub-package documents.

Reply:
I wasn't really sure what this section in the SEP markdown meant- but I am no longer combining files in this way.